CHANGELOG.md

```diff
@@ -10,17 +10,27 @@
 ### Bug Fixes 
 
 - Flag added to toggle `eth/65` off by default. `eth/65` will remain toggled off by default until 
-a fix is completed for the [eth/65 known issue](KNOWN_ISSUES.md). 
+a fix is completed for the [eth/65 known issue](KNOWN_ISSUES.md). [\#741](https://github.com/hyperledger/besu/pull/741)
+- Resolve crashing NAT detectors on GKE. [\#731](https://github.com/hyperledger/besu/pull/731) fixes [\#507](https://github.com/hyperledger/besu/issues/507). 
+[Besu-Kubernetes Readme](https://github.com/PegaSysEng/besu-kubernetes/blob/master/README.md#network-topology-and-high-availability-requirements) 
+updated to reflect changes.  
+- Deal with quick service start failures [\#714](https://github.com/hyperledger/besu/pull/714) fixes [\#662](https://github.com/hyperledger/besu/issues/662) 
 
 ### Known Issues 
 
 Known issues are open issues categorized as [Very High or High impact](https://wiki.hyperledger.org/display/BESU/Defect+Prioritisation+Policy).
+
+New known issues: 
+
+- `Intrinsic gas exceeds gas limit` returned when calling `delete mapping[addr]` or `mapping[addr] = 0` [\#696](https://github.com/hyperledger/besu/issues/696)
+
+Calling delete and set to 0 Solidity mapping in Solidity fail.
+
 Previously identified known issues: 
 
 - [Eth/65 not backwards compatible](KNOWN_ISSUES.md#eth65-not-backwards-compatible)
```

Comment:
Should we link mitigations?  For this https://github.com/hyperledger/besu/pull/741 disabled ETH/65 by default.

Reply:
Probably, yes, but perhaps in `KNOWN_ISSUES.md` ? I think we can add that PR to this line: https://github.com/hyperledger/besu/pull/754/files#diff-05319f6cacd1bf22bdbbf49f484df078L16

Reply:
Added in known issues.  741 is also listed in the bug fixes.